x11-misc/redshift-qt/Manifest

```diff
@@ -0,0 +1,5 @@
+DIST v0.3.tar.gz 171365 BLAKE2B 7789d9be8b55f16ea8248f5455d9ce5653a9e4e280f8ddec172dcf693acf9a631a11dd41caf54b6ead95fdb911888354e6e48dcea810fd97b7501c290e1f0714 SHA512 17cc0c03cb55be7383a67f6d4698bdcf3bcee00d2fe7c7db9fb7b1ab774531d531528df3cba8c2c5af81243a8b90655575eedb9885a6b02d166059942f4e85fc
+DIST v0.4.tar.gz 171732 BLAKE2B 1822326fd924b17033b640261346566426c4030bce0252e79ee7a4840b7a0c1c038a71c3e399eaeb6b1646d3daf40eed920bc5d3e13fdd21e8fbc7b2699ebe2d SHA512 18c9aef516a4bd84c298128d3867d1de946f6a184fb250f9eb7928474df74d76b2203e2ad78d1766262a4d42b01dc108acc332026f0cd73f42cdc2a78a8f09a2
+EBUILD redshift-qt-0.3.ebuild 733 BLAKE2B c23086b9b340b45509ff656a132e450befcf3e1bd5133f2e77d046bf0e0be076962a89cf092dd9e976742d2725e097df61dba58a733b5d5272b8e87ae2709d84 SHA512 6bcd2cc826867512020932e35a5f7ee1e86009d8b386a80923d8a929ac8952ec7cb43f91a34f51e486265074868363529ff0fd633f7c9c0f32aae827d48a898f
```

Comment:
please reconsider adding 0.3, what for, when we have 0.4?

Reply:
upstream lists 0.4 as a "pre-release" but it runs fine for me.